app/forms/recovery_code_form.rb

```diff
@@ -21,6 +21,8 @@ def submit
   attr_reader :user, :success
 
   def valid_recovery_code?
+    word_regexp = /\w{#{RandomPhrase::WORD_LENGTH}}/
+    return false unless code =~ /\A#{word_regexp} #{word_regexp} #{word_regexp} #{word_regexp}\Z/
```

Comment:
oh huh! so if they already cover it and they were passing before, I am confused about what behavior this PR is changing?